File: C_Programs/Program2.c
#include<stdio.h>

void Display(int);

int main()
{
	int ino = 0;
	printf("Enter Number ");
	scanf("%d",&ino);
	Display(ino);
	return 0;
	
}

void Display(int iValue)
{
	int i = 0;
	for(i = 1; i<= iValue; i++)
	{
		printf("Numbers is : %d \n",i);
	}
}

File: Java_Programs/Program2.java
import java.util.Scanner;

class Program2
{
	public static void Display(int iValue)
	{
		int i = 0;
		for(i = 1; i <= iValue; i++)
		{
			System.out.println(i);
		}
	}
	
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		
		int ino = 0;
		
		System.out.println("Enter Number");
		ino = sobj.nextInt();
		
		Display(ino);
		
	}
}